.github/workflows/ci.yml

```diff
@@ -928,7 +928,7 @@ jobs:
     strategy:
  #     fail-fast: false
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, macos-latest]
```

Comment:
is this needed? We do replicate the issue with the code on `master`? if that is not the case, I think that we should not add this

Reply:
I added it to check on compilation issues with macos, by building `smartmodules/examples`

Reply:
Yes, but did you replicate the issue with master? I see that CI for macos is failing due to changes not related to this PR

Reply:
You are right! Applied [here](https://github.com/infinyon/fluvio/pull/3275/commits/a01add23578c349133adcb2d6557e9484d8a56a7)